Makefile

```diff
@@ -126,68 +126,59 @@ delete-all-items: ##@dynamodb deletes and recreates lpas dynamodb table
+	curl -X POST "localhost:9001/emit/opg.poas.lpastore/lpa-updated" \
+		-H "Content-Type: application/json" \
+		-d '{"uid": "'$${uid}'", "changeType":"'$${changeType}'"}'
 
-	docker compose -f docker/docker-compose.yml exec localstack awslocal lambda invoke \
-		--endpoint-url=http://localhost:4566 \
-		--region eu-west-1 \
-		--function-name event-received text \
-		--payload '{"Records": [{"messageId": "19dd0b57-b21e-4ac1-bd88-01bbb068cb78", "body": "$(BODY)"}]}'
 
 emit-priority-correspondence-sent: ##@events emits a priority-correspondence-sent event with the given LpaUID e.g. emit-priority-correspondence-sent uid=abc-123
-	$(eval BODY := $(shell echo '{"version":"0","id":"63eb7e5f-1f10-4744-bba9-e16d327c3b98","detail-type":"priority-correspondence-sent","source":"opg.poas.sirius","account":"653761790766","time":"2023-08-30T13:40:30Z","region":"eu-west-1","resources":[],"detail":{"uid":"$(uid)","sentDate":"2024-01-02T12:13:14.000006Z"}}' | sed 's/"/\\"/g'))
+	curl -X POST "localhost:9001/emit/opg.poas.sirius/priority-correspondence-sent" \
+		-H "Content-Type: application/json" \
+		-d '{"uid": "'$${uid}'", "sentDate":"2024-01-02T12:13:14.000006Z"}'
 
-	docker compose -f docker/docker-compose.yml exec localstack awslocal lambda invoke \
-		--endpoint-url=http://localhost:4566 \
-		--region eu-west-1 \
-		--function-name event-received text \
-		--payload '{"Records": [{"messageId": "19dd0b57-b21e-4ac1-bd88-01bbb068cb78", "body": "$(BODY)"}]}'
+emit-immaterial-change-confirmed: ##@events emits a immaterial-change-confirmed event with the given LpaUID, actor type an actor UID e.g. emit-immaterial-change-confirmed uid=abc-123 actorType=donor actorUid=def-456
+	curl -X POST "localhost:9001/emit/opg.poas.sirius/immaterial-change-confirmed" \
+		-H "Content-Type: application/json" \
+		-d '{"uid": "'$${uid}'", "actorType": "'$${actorType}'", "actorUID": "'$${actorUID}'"}'
+
+emit-material-change-confirmed: ##@events emits a material-change-confirmed event with the given LpaUID, actor type an actor UID e.g. emit-material-change-confirmed uid=abc-123 actorType=donor actorUid=def-456
+	curl -X POST "localhost:9001/emit/opg.poas.sirius/material-change-confirmed" \
+		-H "Content-Type: application/json" \
+		-d '{"uid": "'$${uid}'", "actorType": "'$${actorType}'", "actorUID": "'$${actorUID}'"}'
+
+emit-certificate-provider-identity-check-failed: ##@events emits a certificate-provider-identity-check-failed event with the given LpaUID e.g. certificate-provider-identity-check-failed uid=abc-123
+	curl -X POST "localhost:9001/emit/opg.poas.sirius/certificate-provider-identity-check-failed" \
+		-H "Content-Type: application/json" \
+		-d '{"uid": "'$${uid}'"}'
+
+format-event: ##@events prints an event in the correct format for use testing lambdas in AWS console e.g. format-event detailType=uid-requested detail='{"uid":"abc"}'
```

Comment:
This is useful for throwing stuff into lambdas without having to remember the format